src/config/env.rs

```diff
@@ -0,0 +1,84 @@
+use std::str::FromStr;
+
+use super::builder::NotDetermined;
+use crate::{DeviceConfig, DeviceError, DeviceResult};
+
+#[derive(Debug)]
+enum Source {
+    Env(String),
+    Try(String),
```

Comment:
In my opinion, the expression `Try` is a bit unclear.
In the implementation, both `Env` and `Try` are trying the same way. (`DeviceConfig::from_str()`) `Try` doesn't seem to mean 'source'.
How about changing `Try` to `Str`? It will be used like this:
```rust
DeviceConfig::from_env("ENV_KEY_1").or_env("ENV_KEY_2").or_str("0:0,0:1").build();
```
`or_str` may correspond to `DeviceConfig::from_str`.

Reply:
Agree. I applied it in [18be0aa](https://github.com/furiosa-ai/device-api/pull/62/commits/18be0aa085d763cb3d29ea20daf1877078a43039).